yarn.lock

```diff
@@ -41,9 +41,9 @@
     pretty-ms "^5.0.0"
 
 "@electron/typescript-definitions@^8.6.1":
-  version "8.6.1"
-  resolved "https://registry.yarnpkg.com/@electron/typescript-definitions/-/typescript-definitions-8.6.1.tgz#7d7b506c21d6667f9e53e5930e3b70a9c1d4ccf3"
-  integrity sha512-lv6tj76wnhWzTHuxryhTV329AWHPoOn+VUpCzsz3XVCKm+FO+ldttdgFv+1lA7YGOsi+ttiaX678g+iBGZDlTQ==
+  version "8.6.3"
```

Comment:
was this an intentional change?

Reply:
Yes.  version 8.6.3 is needed to make this doc change.